### ISSUE TITLE
<answer>s go outside of <statement>s

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/precalculus#121]
We should have caught this before approving #112, but the print is currently not building because section FN-3 has a bunch of `<answer>`s that are inside of `<statement>`s.  These should go immediately after the `<statement>`.

**Comment by StevenClontz** [comment]
Yeah I think that green checkmark is too lenient right now. It needs to be better about raising a flag if there are errors in a build.

**Comment by siwelwerd** [comment]
I think this is an upstream issue with PreTeXt, really.  Here's what I see in the output for the build:

```
! Undefined control sequence.
l.3410 \noindentChoice
                       C: \([-5,4]\)%
Output written on main.pdf (144 pages).
Transcript written on main.log.

Success! Run `pretext view` to see the results.
```

LaTeX is failing, but in a way that produces a(n aborted) PDF, and PreTeXt is interpreting that as "Success!".


Much higher up, I see PreTeXt detecting something amiss but happily churning along anyways:
```
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
    * PTX:BUG:     Exercise component mis-labeled
  ```

**Comment by siwelwerd** [comment]
By the way, I noticed this morning that `pretext build web` does not report the PTX:BUG, it only shows up when building a latex output like `print` or `slides`.

**Comment by StevenClontz** [comment]
Oh I just hit the `\noindentWORDS` bug in another project I helped with earlier today: $5 says there's a numbered element like an activity in an `<introduction/>` somewhere. This isn't allowed in the PreTeXt schema, and breaks the PDF build, but happens to not break HTML builds since HTML is a more lenient markup.

**Comment by StevenClontz** [comment]
Oh I guess I lose $5 because I scrolled up and remembered the context. Anyway, yeah, I need to help improve this upstream.

**Comment by tdegeorge** [comment]
Just submitted a pull request. I believe I fixed all the answer/statement issues in FN3

**Comment by siwelwerd** [comment]
Closed by #131
